tests/test_background_worker.py

```
@@ -0,0 +1,473 @@
+
+    def test_validate_localhost_blocked(self):
+        """Should block localhost and loopback addresses."""
+        localhost_urls = [
+            'http://localhost/test',
+            'http://127.0.0.1/api',
+            'http://[::1]/admin'
+        ]
+
+        for url in localhost_urls:
+            with patch('socket.gethostbyname', return_value='127.0.0.1'):
+                valid, reason, _ = _validate_and_resolve_target(url)
```

Ruff (0.14.10)

Unpacked variable `reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)